diff --git a/src/arr/compiler/cli-module-loader.arr b/src/arr/compiler/cli-module-loader.arr
index 179437f0e..f691a1d27 100644
--- a/src/arr/compiler/cli-module-loader.arr
+++ b/src/arr/compiler/cli-module-loader.arr
@@ -374,6 +374,9 @@ fun module-finder(ctxt :: CLIContext, dep :: CS.Dependency):
         else:
           raise("Cannot find import " + torepr(dep))
         end
+      else if protocol == "file-reset-load-path":
+        new-context = ctxt.{current-load-path: "."}
+        module-finder(new-context, CS.dependency("file", args))
       else if protocol == "js-file":
         clp = ctxt.current-load-path
         real-path = get-real-path(clp, args.get(0))
