diff --git a/src/arr/compiler/cli-module-loader.arr b/src/arr/compiler/cli-module-loader.arr
index 179437f0e..a6e86c98e 100644
--- a/src/arr/compiler/cli-module-loader.arr
+++ b/src/arr/compiler/cli-module-loader.arr
@@ -551,11 +551,9 @@ fun build-runnable-standalone(path, require-config-path, outfile, options) block
     | left(problems) =>
       handle-compilation-errors(problems, options)
     | right(program) =>
-      shadow require-config-path = if not( Filesystem.is-absolute( require-config-path ) ):
-          Filesystem.resolve(Filesystem.join(options.base-dir, require-config-path))
-        else: require-config-path
-        end
-      config.set-now("out", JSON.j-str(Filesystem.resolve(Filesystem.join(options.base-dir, outfile))))
+      shadow require-config-path = get-real-path(options.base-dir, require-config-path)
+
+      config.set-now("out", JSON.j-str(get-real-path(options.base-dir, outfile)))
       when not(config.has-key-now("baseUrl")):
         config.set-now("baseUrl", JSON.j-str(options.compiled-cache))
       end
