diff --git a/src/js/trove/require-util.js b/src/js/trove/require-util.js
index a82db7776..36852b349 100644
--- a/src/js/trove/require-util.js
+++ b/src/js/trove/require-util.js
@@ -13,13 +13,17 @@
             runtime.checkArgsInternal2('require-util', 'resolve', moduleName, runtime.String, baseDir, runtime.String);
             try {
                 return runtime.pauseStack((restarter) => {
-                  browserifyResolve(moduleName, { basedir: baseDir }, (err, resolved) => {
-                      if(err) { restarter.error(runtime.makeMessageException(`Error resolving ${moduleName} from ${baseDir}: ${String(err)}`)); }
-                      restarter.resume(resolved);
-                  });
+                    browserifyResolve(moduleName, { basedir: baseDir, preserveSymlinks: true }, (err, resolved) => {
+                        if (err) {
+                            console.dir(err);
+                            restarter.error(runtime.makeMessageException(`[CB] Error resolving ${moduleName} from ${baseDir}: ${String(err)}`));
+                        }
+                        restarter.resume(resolved);
+                    });
                 });
             } catch (err) {
-                throw runtime.throwMessageException(`Error resolving ${moduleName} from ${baseDir}: ${String(err)}`);
+                console.dir(err);
+                throw runtime.throwMessageException(`[Catch] Error resolving ${moduleName} from ${baseDir}: ${String(err)}`);
             }
         }
         function cannotResolve(moduleName) {
